routes/leaguesearch.js

```diff
@@ -156,8 +157,9 @@ var createTeamLists = function(teamobject){
 var sortTeams = function(teamsobject){
   for (var key in teamsobject){
     teamsobject[key].sort(compareCounts);
+    var limited = teamsobject[key].slice(0,10);
```

Comment:
why can't we limit in mongo? Ideally we would do it there so it saves us db work.